package.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "version": "1.0.0",
   "scripts": {
-    "start": "vite",
+    "start": "vite --host",
```

Comment:
What does this thing do? 🤔

Reply:
enable address for local wi-fi that helps test on mobile)
<img width="328" alt="Screenshot 2022-11-24 at 18 50 21" src="https://user-images.githubusercontent.com/4715396/203850168-309c084f-567f-4abf-a613-ba63fa824232.png">

Reply:
Lets add to `vite.config.js`

export default defineConfig({
  server: {
    host: true
  },
  build: {
    assetsDir: '.'
  }
})

https://github.com/ekaterinburgdev/ekaterinburg.dev/blob/main/vite.config.js#L4

Reply:
done